checkov/kustomize/runner.py

```diff
@@ -171,19 +174,45 @@ def check_system_deps(self):
         # Ensure local system dependancies are available and of the correct version.
         # Returns framework names to skip if deps **fail** (ie, return None for a successful deps check).
         logging.info(f"Checking necessary system dependancies for {self.check_type} checks.")
-        try:
-            proc = subprocess.Popen([self.kustomize_command, 'version'], stdout=subprocess.PIPE, stderr=subprocess.PIPE)  # nosec
-            o, e = proc.communicate()
-            oString = str(o, 'utf-8')
-
-            if "Version:" in oString:
-                kustomizeVersionOutput = oString[oString.find('/') + 1: oString.find('G') - 1]
-                logging.info(f"Found working version of {self.check_type} dependancies: {kustomizeVersionOutput}")
-                return None
-            else:
-                return self.check_type
-        except Exception:
-            logging.info(f"Error running necessary tools to process {self.check_type} checks.")
+
+        if shutil.which(self.kubectl_command) != None:
```

Comment:
```suggestion
        if shutil.which(self.kubectl_command) is not None:
```

Reply:
Thanks! Changed as the linter picked it up! :)